documentation/mkdocs.yml

```diff
@@ -102,6 +102,7 @@ plugins:
         version_selector: true
         css_dir: css
         javascript_dir: javascript
+  - print-site
```

Comment:
what does it do?

Reply:
It creates a printable single page doc in path `/print_page`